content/source/docs/enterprise/api/policies.html.md

```diff
@@ -68,7 +69,7 @@ Key path                         | Type            | Default          | Descript
           "mode": "hard-mandatory"
         }
       ],
-      "name": "my-example-policy",
```

Comment:
Good catch, and also I've _got_ to figure out a way to make this less difficult. Could probably rig up an editor plugin somehow.........